src/templates/next/components/complex/Accordion/Accordion.tsx

```diff
@@ -4,8 +4,8 @@ import { Prose } from "../../native"
 
 const Accordion = ({ summary, details }: AccordionProps) => {
   return (
-    <details className="[&_>_summary]:open:font-semibold [&_>_summary_>_svg]:open:rotate-180 border-y has-[+_details]:border-b-0 border-divider-medium">
-      <summary className="my-2 px-4 py-3 flex flex-row gap-3 outline-none list-none text-content focus:ring-2 focus:ring-offset-0 focus:ring-focus-outline">
+    <details className="[&_>_summary]:open:font-semibold [&_>_summary_>_svg]:open:rotate-180 border-y has-[+_details]:border-b-0 border-divider-medium hover:cursor-pointer">
+      <summary className="my-[0.125rem] px-4 py-5 flex flex-row gap-3 outline-none list-none text-content focus:ring-2 focus:ring-offset-0 focus:ring-focus-outline">
```

Comment:
issue: need to use the Paragraph-1 style here, otherwise the summary will be 16px but the details body is at 20px

Reply:
sure, can add para 1, but this comment is not true right? rn in storybook both 16 leh
<img width="852" alt="Screenshot 2024-05-14 at 3 36 14 PM" src="https://github.com/isomerpages/isomer-components/assets/42832651/41be7dda-0e50-40cf-88a9-17f77728c455">
<img width="843" alt="Screenshot 2024-05-14 at 3 36 08 PM" src="https://github.com/isomerpages/isomer-components/assets/42832651/629a267d-1740-4fb1-8099-d53422c670d3">

Reply:
Hmm no, right now the details body is at 20px but summary is at 16px, both needs to be standardised to be at 20px which is paragraph-1

Reply:
arh was a breakpoints thing, regardless have added para 1 styles to the <summary> tag already bah